Makefile

```diff
@@ -293,9 +298,11 @@ static-check: gocyclo govet importcheck
 goimports:
 	goimports -w $(GOFMTFILES)
 
+GOPATH=$(shell go env GOPATH)
 .get-deps-stamp:
 	go get golang.org/x/tools/cmd/cover
 	go get github.com/golang/mock/mockgen
+	cd "${GOPATH}/src/github.com/golang/mock/mockgen" && git checkout 1.3.1 && go get ./... && go install ./...
```

Comment:
do we need to cd back to previous directory here? wondering if this will affect other target if we don't cd back

Reply:
sure, I can change that